.github/workflows/create-preview-branch.yml

```diff
@@ -14,10 +14,11 @@ concurrency:
 permissions:
   contents: write    # Required to checkout, create branches, and push
   actions: read      # Required to run the workflow
-  pull-requests: write  # Required to comment on PRs
 
 jobs:
   create-preview:
+    permissions:
+      pull-requests: write # Required to comment on PRs
```

Comment:
@eyurtsev I moved this to the job level because I didn't think there was a way to set permissions on an individual step. Is this what you were thinking or was there a different solution?

Reply:
Not sure this change is the cause yet since it didn't fail on the step for commenting on the PR but the preview workflow failed: https://github.com/langchain-ai/docs/actions/runs/16992739025/job/48175935263?pr=169#step:11:892